server/src/tsp-client.ts

```diff
@@ -10,10 +10,9 @@ import * as path from 'path';
 import * as cp from 'child_process';
 import * as readline from 'readline';
 import * as decoder from 'string_decoder';
-import protocol from 'typescript/lib/protocol';
+import tsp from 'typescript/lib/protocol';
```

Comment:
I'd suggest this instead:
```suggestion
import protocol, { CommandTypes } from 'typescript/lib/protocol';
```
Then you avoid all unnecessary changes and the diff is minimal.

Similarly in other files.